demos/999-storage-import-vhd-blob/prereqs/CreateVHDBlobUri.ps1

```diff
@@ -0,0 +1,32 @@
+param(
```

Comment:
remove this file and use one of the following:

  "GEN-SPECIALIZED-WINVHD-URI"
  "GEN-GENERALIZED-WINVHD-URI"
  "GEN-GENERALIZED-UBUNTUVHD-URI"

Reply:
done (awesome)